apis/validator/block.v3.yaml

```diff
@@ -43,6 +43,39 @@ get:
         is a flag and does not take a value.
       schema: {}
       allowEmptyValue: true
+    - name: builder_boost_factor
+      in: query
+      required: false
+      description: |
+        Percentage multiplier to apply to the builder's payload value when choosing between a
+        builder payload header and payload from the paired execution node. This parameter is only
+        relevant if the beacon node is connected to a builder, deems it safe to produce a builder
+        payload, and receives valid responses from both the builder endpoint _and_ the paired
+        execution node. When these preconditions are met, the server MUST act as follows:
+
+        * if `exec_node_payload_value >= builder_comparison_factor * (builder_payload_value // 100)`,
+          then return a full (unblinded) block containing the execution node payload.
+        * otherwise, return a blinded block containing the builder payload header.
+
+        Servers must support the following values of the comparison factor which encode common
+        preferences:
+
+        * `builder_comparison_factor=0`: prefer the execution node payload unless an error makes it
+          unviable.
```

Comment:
What does this mean? if the beacon health check makes it unviable, should we return the local payload or error out?